Azure release 11.0.0 with Kubernetes 1.16.3 (#62)
giantswarm/releases: release-notes/azure/v11.0.0.md

## :zap: Giant Swarm Release 11.0.0 for Azure is now active for you! :zap:

This is the first Giant Swarm release which includes Kubernetes v1.16. In addition, CPU limits have been removed from several supporting components and priority classes have been adjusted to ensure system reliability under heavy load. Further details about changes to individual components can be found below.

### Important upgrade notes

> - This release includes a new [network policy](https://docs.giantswarm.io/guides/limiting-pod-communication-with-network-policies/#default-policies) which blocks network traffic to and from pods in the `giantswarm` and `kube-system` namespaces by default to improve security. GS components have been modified to work under this environment, but any other pods in the cluster communicating with these system pods may cease to function without a network policy allowing traffic.

Reviewer: ```suggestion
- This release includes a new [network policy](https://docs.giantswarm.io/guides/limiting-pod-communication-with-network-policies/#default-policies) which blocks network traffic to and from pods in the `giantswarm` and `kube-system` namespaces by default to improve security. GS components have been modified to work under this environment, but any other pods in the cluster communicating with these system pods may cease to function without a network policy allowing traffic. Thus, if you are running pods that need to talk to system pods, you will need to allow those explicitly by adding a [network policy](https://docs.giantswarm.io/guides/limiting-pod-communication-with-network-policies/) for them.
```

> - [As previously communicated](https://github.com/giantswarm/giantswarm/blob/master/news/2019/10/product/k8s_1.16_breaking_changes.md), Kubernetes v1.16 has removed several API groups which were already deprecated. Please ensure that all affected components have been updated to the 1.16 compatible versions before upgrading clusters from v1.15.

Reviewer: ```suggestion
- [As previously communicated](https://github.com/giantswarm/giantswarm/blob/master/news/2019/10/product/k8s_1.16_breaking_changes.md), Kubernetes v1.16 has removed several API groups which were already deprecated. Please ensure that you have migrated your deployments and pipelines from the deprecated to the new API groups. You can check the above-linked breaking change communication and consult with your Solution Engineer for help with that.
```

> - Previous Giant Swarm releases included Tiller v2.14 in the `giantswarm` namespace which is not compatible with Kubernetes v1.16. This Tiller used by Giant Swarm to manage the cluster has therefore been updated to the Kubernetes v1.16-compatible v2.16. Any Tiller instances deployed in other namespaces will likewise need to be updated to v2.16 or greater to continue to function after an upgrade.

Reviewer: ```suggestion
- Previous Giant Swarm releases included Tiller v2.14 in the `giantswarm` namespace which is not compatible with Kubernetes v1.16. This Tiller - used by Giant Swarm to manage the cluster - has therefore been updated to the Kubernetes v1.16-compatible v2.16. Any Tiller instances deployed in other namespaces will likewise need to be updated to v2.16 or greater to continue to function after an upgrade.
```

## :zap: Giant Swarm Release 11.0.0 for Azure is now active for you! :zap:

> This is the first Giant Swarm release including Kubernetes 1.16. As part of this release, we have also removed CPU limits from several supporting components and adjust priority classes to ensure system reliability under heavy load. More details about changes to individual components can be found below.

Author: ```suggestion
This is the first Giant Swarm release including Kubernetes 1.16. As part of this release, we have also removed CPU limits from several supporting components and adjusted priority classes to ensure system reliability under heavy load. More details about changes to individual components can be found below.
```

> **Note**: This release includes a [network policy](https://docs.giantswarm.io/guides/limiting-pod-communication-with-network-policies/#default-policies) which blocks traffic to and from pods in the `giantswarm` and `kube-system` namespaces to improve security. GS components have modified to work under this environment, but any other workload pods in these namespaces may cease to function without a network policy allowing ingress and egress.

Author: ```suggestion
**Note**: This release includes a [network policy](https://docs.giantswarm.io/guides/limiting-pod-communication-with-network-policies/#default-policies) which blocks traffic to and from pods in the `giantswarm` and `kube-system` namespaces to improve security. GS components have been modified to work under this environment, but any other workload pods in these namespaces may cease to function without a network policy allowing ingress and egress.
```
